macos/Flutter/ephemeral/Flutter-Generated.xcconfig

```diff
@@ -1,10 +1,10 @@
 // This is a generated file; do not edit or check into version control.
-FLUTTER_ROOT=/Users/sanketh.bk/sdk/flutter3
-FLUTTER_APPLICATION_PATH=/Users/sanketh.bk/my_projects/flutter/apps in production/dairy_app
+FLUTTER_ROOT=/Users/user/Documents/src/flutter
+FLUTTER_APPLICATION_PATH=/Users/user/Documents/Projects/Solo Project/diaryvault
 COCOAPODS_PARALLEL_CODE_SIGN=true
```

Comment:
Remove these 2 files inside macos folder as they are generated files

Reply:
hey @SankethBK can you review again my new commit? thanks.